crates/re_query_cache/src/range.rs

```diff
@@ -171,16 +179,41 @@ macro_rules! impl_query_archetype_range {
             let mut range_callback = |query: &RangeQuery, range_cache: &mut crate::RangeCache| {
                 re_tracing::profile_scope!("range", format!("{query:?}"));
 
-                for reduced_query in range_cache.compute_queries(query) {
+                // NOTE: Same logic as what the store does.
+                if query.range.min <= TimeInt::MIN {
+                    let mut reduced_query = query.clone();
+                    reduced_query.range.max = TimeInt::MIN; // inclusive
```

Comment:
This line looks suspicious to me. Doesn't setting the max to the min make this an empty query?

Reply:
It's inclusive so it yields `MIN..=MIN` = `[MIN]`

Reply:
Got it -- some kind of comment here would be helpful explaining this is the reduced query corresponding to the timeless part of the data?

Reply:
will do